src/genie/libs/parser/iosxe/show_ap.py

```diff
@@ -295,3 +295,782 @@ def cli(self, output=None):
+            elif channel_noise_capture.match(line):
+                channel_noise_capture_match = channel_noise_capture.match(line)
+                groups = channel_noise_capture_match.groupdict()
+                channel_noise = groups['channel_noise']
+                show_ap_dot11_5ghz_channel_dict['channel_assignment'].update({'channel_noise': channel_noise})
+                continue
+            #     Interference                             : Enable
+            elif channel_interference_capture.match(line):
+                channel_interference_capture_match = channel_interference_capture.match(line)
+                groups = channel_interference_capture_match.groupdict()
+                channel_interference = groups['channel_interference']
+                show_ap_dot11_5ghz_channel_dict['channel_assignment'].update(
+                    {'channel_interference': channel_interference})
+                continue
+                #     Load                                     : Disable
+            elif channel_load_capture.match(line):
+                channel_load_capture_match = channel_load_capture.match(line)
+                groups = channel_load_capture_match.groupdict()
+                channel_load = groups['channel_load']
+                show_ap_dot11_5ghz_channel_dict['channel_assignment'].update({'channel_load': channel_load})
+                continue
+            #     Device Aware                             : Disable
+            elif device_aware_capture.match(line):
+                device_aware_capture_match = device_aware_capture.match(line)
+                groups = device_aware_capture_match.groupdict()
+                device_aware = groups['device_aware']
+                show_ap_dot11_5ghz_channel_dict['channel_assignment'].update({'device_aware': device_aware})
+                continue
+            #   CleanAir Event-driven RRM option           : Disabled
+            elif clean_air_capture.match(line):
+                clean_air_capture_match = clean_air_capture.match(line)
+                groups = clean_air_capture_match.groupdict()
+                clean_air = groups['clean_air']
+                show_ap_dot11_5ghz_channel_dict['channel_assignment'].update({'clean_air': clean_air})
+                continue
+            #   Channel Assignment Leader                  : sj-00a-ewlc1 (10.7.5.133)
+            elif chan_assn_leader_capture.match(line):
+                chan_assn_leader_capture_match = chan_assn_leader_capture.match(line)
+                groups = chan_assn_leader_capture_match.groupdict()
+                wlc_leader_name = groups['wlc_leader_name']
+                wlc_leader_ip = groups['wlc_leader_ip']
+                show_ap_dot11_5ghz_channel_dict['channel_assignment'].update({'wlc_leader_name': wlc_leader_name})
+                show_ap_dot11_5ghz_channel_dict['channel_assignment'].update(
+                    {'wlc_leader_ip': wlc_leader_ip.replace('(', '').replace(')', '')})
```

Comment:
If you are only catching the Ip address without the `()` you won't need that string replace call